stories/topics/proc-gcp-backup-requirements.adoc

```diff
@@ -0,0 +1,28 @@
+[id="proc-gcp-backup-requirements"]
+
+= Backup requirements
+
+A bucket must be created in order to store the Ansible on Clouds deployment backup.
+The bucket will contains the backup information, the secrets and the databases backups.
+
+.Procedure
+. Open the Google Cloud console and navigate to the Cloud Storage -> Buckets
++
```

Comment:
You don't need the + between lines for this procedure

Reply:
Done